homeassistant/components/home_connect/__init__.py

```diff
@@ -338,8 +338,8 @@ def get_dict_from_home_connect_error(err: api.HomeConnectError) -> dict[str, Any
         if len(err.args) > 0 and isinstance(err.args[0], dict)
         else err.args[0]
         if len(err.args) > 0 and isinstance(err.args[0], str)
-        else {}
-    )
+        else "?",
```

Comment:
A nested (!) ternary operator expression that spans more than one line. 😄

I'd rewrite this and avoid the ternary operators.